java/src/test/java/com/saucelabs/saucebindings/SauceVisualSession.java

```diff
@@ -0,0 +1,66 @@
+package com.saucelabs.saucebindings;
+
+import com.saucelabs.saucebindings.visual.SauceVisualOptions;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.var;
+import org.openqa.selenium.InvalidArgumentException;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.MutableCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Map;
+
+public class SauceVisualSession {
+    private SauceVisualOptions visualOptions;
+    private RemoteWebDriver driver;
+
+    public SauceVisualSession() {
```

Comment:
Codacy found an issue: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/manual/SauceLabs/sauce_bindings/pullRequest?prid=6239731)